pages/common/nodetool.md

```diff
@@ -0,0 +1,11 @@
+# nodetool
+
+> Command line interface for managing a cassandra cluster.
+
+- Check the health of your cluster:
+
+`nodetool --host {{host}} --username {{username}} --password {{password}} status`
```

Comment:
Specifying a password on the command-line isn't very secure. Is there a way of doing this without doing that?

Reply:
There seems to be a `--password-file` which takes the path to a password file. But I think the tool should prompt for the password if it is not passed explicitly. That is the ideal behavior.